dataset/src/main/scala/frameless/functions/UnaryFunctions.scala

```diff
@@ -28,6 +28,15 @@ trait UnaryFunctions {
     */
   def sortDescending[T, A: Ordering, V[_] : CatalystSortableCollection](column: TypedColumn[T, V[A]]): TypedColumn[T, V[A]] =
     new TypedColumn[T, V[A]](implicitly[CatalystSortableCollection[V]].sortOp(column.untyped, sortAscending = false))(column.uencoder)
+
+
+  /** Creates a new row for each element in the given collection. The column types eligible for this
```

Comment:
Reviewing my own comment, sorry...

*are* constrained by the CatalystExplodableCollection.

While you're at it you might also cut the line at 80 chars to keep it consistent :smile: